modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -214,27 +228,28 @@ platform:
     powervsResourceGroup: "ibmcloud-resource-group"
     region: powervs-region
     vpcRegion : vpc-region
-    vpcName: name-of-existing-vpc <6>
-    vpcSubnets: <7>
+    vpcName: name-of-existing-vpc <7>
+    vpcSubnets: <8>
     - powervs-region-example-subnet-1
     zone: powervs-zone
     serviceInstanceID: "powervs-region-service-instance-id"
 credentialsMode: Manual
-publish: External <8>
-pullSecret: '{"auths": ...}' <9>
+publish: External <9>
+pullSecret: '{"auths": ...}' <10>
 fips: false
-sshKey: ssh-ed25519 AAAA... <10>
+sshKey: ssh-ed25519 AAAA... <11>
 ----
 <1> If you do not provide these parameters and values, the installation program provides the default value.
 <2> The `controlPlane` section is a single mapping, but the compute section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Both sections currently define a single machine pool. Only one control plane pool is used.
 <3> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
-<4> The machine CIDR must contain the subnets for the compute machines and control plane machines.
-<5> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<6> Specify the name of an existing VPC.
-<7> Specify the name of the existing VPC subnet. The subnets must belong to the VPC that you specified. Specify a subnet for each availability zone in the region.
-<8> How to publish the user-facing endpoints of your cluster.
-<9> Required. The installation program prompts you for this value.
-<10> Provide the `sshKey` value that you use to access the machines in your cluster.
+<4> The SMTLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are from 1 to 8.
```

Comment:
You cannot set it to any value between 1 and 8.  For e.g. 3, 5, 6, 7 etc.  Only Powers of 2 are allowed, 1, 2, 4, 8.